api/v1beta1/hyperconverged_types.go

```diff
@@ -19,10 +19,12 @@ import (
 const HyperConvergedName = "kubevirt-hyperconverged"
 
 type HyperConvergedUninstallStrategy string
+type HyperConvergedTuningPolicy string
 
 const (
 	HyperConvergedUninstallStrategyRemoveWorkloads                HyperConvergedUninstallStrategy = "RemoveWorkloads"
 	HyperConvergedUninstallStrategyBlockUninstallIfWorkloadsExist HyperConvergedUninstallStrategy = "BlockUninstallIfWorkloadsExist"
+	HyperConvergedTuningPolicyConfigMap                           HyperConvergedTuningPolicy      = "static-policy"
```

Comment:
please split the `HyperConvergedUninstallStrategy` consts and the `HyperConvergedTuningPolicy` const. Please move the `HyperConvergedTuningPolicy` type definition to be after the const, and add the new const below it.

Reply:
I think I changed it as you requested. Please, let me know.